compiler/qsc_doc_gen/src/generate_docs.rs

```diff
@@ -60,9 +59,9 @@ description: {}
 author: {{AUTHOR}}
 ms.author: {{MS_AUTHOR}}
 ms.date: {{TIMESTAMP}}
-ms.topic: {}
+ms.topic: landing-page
```

Comment:
So landing-page is the only ms.topic that we leave to table of content?

Reply:
Yes, that's right